packages/playground/src/samples/enumObjects.ts

```diff
@@ -1,11 +1,13 @@
 import { Sample } from './Sample';
 
+const ENUM_NAMES = {
+  'ui:enumNames': ['New York', 'Amsterdam', 'Hong Kong'],
+};
+
 const enumObjects: Sample = {
   schema: {
     definitions: {
       locations: {
-        // @ts-expect-error -- enumNames an RJSF keyword and is not in the JSON Schema spec
-        enumNames: ['New York', 'Amsterdam', 'Hong Kong'],
```

Comment:
Is keeping `enumNames` blocking anything? I would rather not remove it if we're losing this functionality (which is essential IMO).

Reply:
Also we mentioned keeping enumNames in v6 https://github.com/rjsf-team/react-jsonschema-form/issues/3829#issuecomment-2231303702 (though that was before it was ported to the UiSchema). 

I think we would need to add the ability to attach a UiSchema to a definition before I would feel comfortable removing this.

Reply:
As we discussed, let's remove custom JSON Schema code to minimize the challenges for supporting newer versions